manim/__init__.py

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env python
+
+# Importing config should be the first thing since other modules use it
+from .config import config
```

Comment:
I don't think this is necessary since the config will be initialized the first time anything imports it.

Reply:
This is here to guarantee that _nothing_ happens before setting the configuration.

I agree it might be unnecessary, but I think it actually doesn't hurt.

Reply:
I'm all for enforcing standards where there's a reason for it, but it feels superstitious to do this without knowing why.

If you see a benefit to always doing it first (e.g. always initializing directories?) we should update the comment with that. As it is it gives the impression that we're trying to do something without saying what it is.

Reply:
I'm ok with this import; after all,
> Explicit is better than implicit.